### DISCUSSION TITLE
2023/07/20/deep-dive-into-kernel-fusion-accelerating-inference-in-llama-v2/

### DISCUSSION THREAD

**Discussion by giscus** [ELS-RD/ai.lefebvre-sarrut#42] [_]
# 2023/07/20/deep-dive-into-kernel-fusion-accelerating-inference-in-llama-v2/

A blog fully dedicated to AI at Lefebvre Sarrut

https://ai.lefebvre-sarrut.eu/2023/07/20/deep-dive-into-kernel-fusion-accelerating-inference-in-llama-v2/

**Comment by purejomo** [comment]
what a nice post!!
I'm waiting for next one!

**Comment by purejomo** [comment]
I'm little bit confused.
The Nsight system profiling reuslt shows the RoPE latency is about 196us, but the posting says the latency has been changed from 0.027ms to 0.006ms 
May I know what is the meaning of 0.0027ms??
